### ISSUE TITLE
MUSCLE: pkcs15-init --create-pkcs15 [Incorrect parameters in APDU]

### ISSUE THREAD

**Issue by amitkumar50** [OpenSC/OpenSC#1789]
### Problem Description
I cannot intialize the card using pkcs15.
Smart card has user certificates installed already.
This can be cleaned
`# lsusb`
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 001 Device 004: ID 5986:0706 Acer, Inc 
Bus 001 Device 003: ID 8087:0a2b Intel Corp. 
Bus 001 Device 002: ID 08e6:3437 Gemalto (was Gemplus) GemPC Twin SmartCard Reader
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub

`# opensc-tool --list-readers`
 Detected readers (pcsc)
Nr.  Card  Features  Name
0    Yes             Gemalto PC Twin Reader 00 00

`# opensc-tool --reader 0 --name`
MuscleApplet

`# opensc-tool -lan`
 Detected readers (pcsc)
Nr.  Card  Features  Name
0    Yes             Gemalto PC Twin Reader 00 00
Using reader with a card: Gemalto PC Twin Reader 00 00
3b:95:95:40:ff:ae:01:03:00:00
MuscleApplet

`# pkcs15-init --create-pkcs15`
Using reader with a card: Gemalto PC Twin Reader 00 00
New User PIN.
Please enter User PIN: 
Please type again to verify: 
Unblock Code for New User PIN (Optional - press return for no PIN).
Please enter User unblocking PIN (PUK): 
Please type again to verify: 
User PIN [User PIN] required.
Please enter User PIN [User PIN]: 
Failed to create PKCS #15 meta structure: Incorrect parameters in APDU

` #pkcs15-init --erase-card --use-default-transport-keys`
Using reader with a card: Gemalto PC Twin Reader 00 00
Unspecified PIN [reference 1] required.
Please enter Unspecified PIN [reference 1]:    <<What should be provided here?

### Steps to reproduce
- Provided above

**Comment by amitkumar50** [comment]
Any updates here?

**Comment by frankmorgner** [comment]
Nobody really maintains the Muscle card applet, please consult the manufacturer of your card how to initialize it

**Comment by Jakuje** [comment]
the muscle is outdated. If you are using different card or applet and it is only misdetected by opensc, please open a new issue.
